plasma_framework/test/endToEndTests/FastExit.e2e.test.js

```diff
@@ -0,0 +1,265 @@
+                            this.transferUtxoPos,
+                        );
+                        const exitIds = [exitId];
+                        const standardExitData = (await this.exitGame.standardExits(exitIds))[0];
+                        const outputIndexForTransfer = 0;
+                        const outputId = computeNormalOutputId(
+                            this.transferTx,
+                            outputIndexForTransfer,
+                            this.transferUtxoPos,
+                        );
+
+                        expect(standardExitData.exitable).to.be.true;
+                        expect(standardExitData.outputId).to.equal(outputId);
+                        expect(new BN(standardExitData.utxoPos)).to.be.bignumber.equal(new BN(this.transferUtxoPos));
+                        expect(standardExitData.exitTarget).to.equal(this.liquidity.address);
+                        expect(new BN(standardExitData.amount)).to.be.bignumber.equal(new BN(TRANSFER_AMOUNT));
+
+                        expect(standardExitData.exitable).to.be.true;
+                    });
+                    it('should not allow to start from the same utxo again', async () => {
+                        const utxoPos = this.transferUtxoPos;
+                        const rlpOutputTx = this.transferTx;
+                        const outputTxInclusionProof = this.merkleProofForTransferTx;
+                        const { depositUtxoPos } = this;
+                        const rlpDepositTx = this.depositTx;
+                        const depositInclusionProof = this.merkleProofForDepositTx;
+
+                        await expectRevert(
+                            this.liquidity.startExit(
+                                utxoPos,
+                                rlpOutputTx,
+                                outputTxInclusionProof,
+                                rlpDepositTx,
+                                depositInclusionProof,
+                                depositUtxoPos,
+                                { from: alice, value: this.startStandardExitBondSize },
+                            ),
+                            'Exit has already started.',
+                        );
+                    });
+
+                    describe('And then Alice processes the exits after two weeks', () => {
+                        before(async () => {
+                            await time.increase(time.duration.weeks(2).add(time.duration.seconds(1)));
+
+                            this.LCBalanceBeforeProcessExit = new BN(await this.liquidity.getContractBalance());
+
+                            await this.framework.processExits(config.registerKeys.vaultId.eth, ETH, 0, 1, {
+                                from: alice,
+                            });
+                        });
+
+                        it('should return the output amount plus standard exit bond to the Liquidity Contract', async () => {
```

Comment:
shouldn't the exit bond be returned to Alice instead?
However, I do think this would need either change to existing contract or add some extra feature in the liquidity contract, could/should be out of scope of this PR. Left a comment on the issue tracking that somewhere though.

Reply:
Yes @boolafish, we could also return it to Alice after the contract receives it.(Alice pulls it)